02/components/App.js

```diff
@@ -9,8 +10,11 @@ class App extends React.Component {
 
     render() {
         const { text } = this.state;
+        const { Provider } = TextContext;
 
-        return <Box text={text} />;
+        return (
+            <Provider value={text}><Box text={text}/></Provider>
+        )
```

Comment:
Jeśli `Provider` przekazuje dane to tutaj już `props` nie jest potrzebny czyli powinno być: `<Provider value={text}><Box/></Provider>`